### PR TITLE
Release 1.0.0

### PR DESCRIPTION
**Pull request by tonyskapunk**

### CODE REVIEW

File: util/packaging/rpm/recap.spec [hunk 1]
```diff
@@ -47,9 +47,26 @@ DESTDIR=%{buildroot} make install
...
+- Install recaptool man page.
+- Adding specific requirements for bash(>=4) and sysstat(>=9).
+- Removing rpm spec requirements: elinks.
+- Obsoletes and provides rs-sysmon.
+- Better support for multiple mysql instances.
+- Makefile updated to allow multiple distros alignment and to include new man pages.
+- Adjust spec for Makefile.
+- Makefile now uses /usr/local as default for DESTDIR.
+- fdisk warnings(stderr) are now included in the logs(stdin).
+
```
Comment (carlwgeorge) [inline util/packaging/rpm/recap.spec]: The goal of the RPM changelog isn't to duplicate the upstream changelog, but rather point out packaging changes.  This entry should just be shortened to:

```
- Latest version
- Include recaptool man page
- Drop elinks requirement
- Add psmisc requirement
- Require minimum versions of bash (4) and sysstat (9)
```

You could even combine the last three into a generic `- Clean up requirements` entry.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,19 +1,21 @@
...
-## [Unreleased]
+## [1.0.0] - 2017-05-15
 - Record point-in-time CPU.
 - Multiple bug fixing on recaptool.
 - Multiple bug fixing on recap.
 - Refactor of recaplog.
 - Fix plesk mysql bug.
 - Install recaptool man page.
 - Adding specific requirements for bash(>=4) and sysstat(>=9).
+- Removing rpm spec requirements: elinks.
```
Comment (carlwgeorge) [inline CHANGELOG.md]: See my other comment regarding RPM changelog versus the software's changelog.  This line should probably just be removed.

**Comment by tonyskapunk** [comment]
@carlwgeorge I've updated the changelogs as pointed out, this PR now includes your commit for the `Makefile` I'm also including the removal of the installer.

**Comment by carlwgeorge** [comment]
I have built this in [COPR](https://copr.fedorainfracloud.org/coprs/carlgeorge/recap/) for early access while I deal with the Fedora/EPEL red tape.

**Comment by carlwgeorge** [comment]
This is currently available in Fedora's updates-testing repo and EPEL's epel-testing repo.  They have already met the time and/or karma requirements and are pending release to the stable repos.

https://bodhi.fedoraproject.org/updates/?packages=recap

**Pull request merged**